MAUI/Kanban-Board/Column.md

```diff
@@ -93,7 +93,7 @@ progressColumn.Categories = new List<object>() { "In Progress" };
 
 ## Headers
 
-Header shows the category [`Title`](), items count, min and max informations of a column. The UI of the header can be replaced entirely using [`HeaderTemplate`]() property of [`SfKanban`](). The following code snippet and screenshot illustrates this.
+Header shows the category `Title`, `items count`, `min and max` informations of a column. The UI of the header can be replaced entirely using [`HeaderTemplate`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Kanban.SfKanban.html#Syncfusion_Maui_Kanban_SfKanban_HeaderTemplate) property of [`SfKanban`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Kanban.SfKanban.html). The following code snippet and screenshot illustrates this.
```

Comment:
@Sowndharya1911  modify as Header shows the category `Title`, `Items count`, `Min` and `Max`